src/components/Table.tsx

```
@@ -0,0 +1,71 @@
+export function Table({ data,tableHead,children }: TableProps) {
+	
+	return (
+		<div className="h-full w-full">
+			<div className="flex flex-col h-full w-full overflow-x-auto">
+				<table className="w-full ">
+					<thead>
+						<tr className="border-b  border-[#1d1d1d]  bg-[#fa9e1f] text-left text-sm leading-4 text-[#000] font-bold h-11  ">
+							{tableHead && tableHead.map((value:string,index:number)=>(
+								<th key={index} className="px-2">{value}</th>
+
+							))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)